docs/dev/lsp-extensions.md

```diff
@@ -1,5 +1,5 @@
 <!---
-lsp_ext.rs hash: 149a5be3c5e469d1
+lsp_ext.rs hash: 121482ee911854da
```

Comment:
`lsp_ext` seem to contain all data wrappers, needed for the resolve operations, hence the hash update.
I can move away both completion and hints structs away, but please propose the new module.